src/main/java/org/apache/maven/plugins/war/util/PathSet.java

```diff
@@ -53,14 +52,14 @@ static String normalizeSubPath( String path )
         {
             return path;
         }
-        String cleanPath = path.replaceAll( "[\\\\]+", SEPARATOR )
-                .replaceAll( "[/]+" , SEPARATOR );
-        cleanPath = cleanPath.charAt( 0 ) == '/' ? cleanPath.substring( 1 ) : cleanPath;
+        String cleanPath = path.replaceAll( "[\\\\]+", File.separator )
```

Comment:
sorry for late reply.
I was meaning mostly the opposite

`String cleanPath = path.replace(File.pathSeparator, "/")`

This will replace windows/linux stuff with a fixed separator ('/')

Reply:
Sorry, but this won't work. path.replace( File.pathSeparator, "/" ) will replace ';' with '/' and ':' with '/'.

Reply:
Sorry for typo, I meant File.separator

Reply:
I forgot to cite this link https://stackoverflow.com/questions/5971964/file-separator-or-file-pathseparator for reference.